tests/e2e/cypress/integration/index.spec.js

```diff
@@ -1,25 +1,44 @@
 describe("Index page spec", () => {
-    it("Gives user transit provider options", () => {
-        cy.visit("/")
-
-        cy.contains("Choose your transit provider")
-          .siblings(".btn")
-          .should('not.be.empty')
-          .each(($e) => {
-              expect($e).attr("href").to.match(/\/[a-z]{3,}$/)
-          })
-    })
-
-    it("Takes user to transit provider page", () => {
-        cy.visit("/")
-
-        cy.get(".buttons .btn")
-          .first()
-          .click()
-
-        cy.contains("Let’s do it!")
-          .then(($e) => {
-            expect($e).attr("href").to.match(/\/eligibility\/$/)
-          })
-    })
+  it("Gives user transit provider options", () => {
+    cy.visit("/")
+
+    cy.contains("Choose your transit provider")
+      .siblings(".btn")
+      .should('not.be.empty')
+      .each(($e) => {
+        expect($e).attr("href").to.match(/\/[a-z]{3,}$/)
+      })
+  })
+
+  it("Takes user to transit provider page", () => {
+    cy.visit("/")
+
+    cy.get(".buttons .btn")
+      .first()
+      .click()
+
+    cy.contains("Let’s do it!")
+      .then(($e) => {
+        expect($e).attr("href").to.match(/\/eligibility\/$/)
+      })
+  })
+
+  it("Has a payment options page", () => {
+    cy.visit("/")
+
+    cy.contains("Payment Options")
+      .click()
+
+    cy.contains("The contactless symbol is four curved lines")
```

Comment:
Let's test the URL, the "Payment Options" heading, and that a button exists that takes user back to index

Reply:
To be resolved in different PR.

Reply:
Was resolved in [#176](https://github.com/cal-itp/benefits/pull/176/files?diff=unified&w=0#diff-346f1b6777a6139ef6aed803c8ff4396f25af2c29eb5f64b872de453443e8920)
 - checked URL
 - checked heading
 - checked Back button